kernel/data-pipeline/scenario/cdc/client/pom.xml

```diff
@@ -45,14 +45,17 @@
         <dependency>
             <groupId>org.opengauss</groupId>
             <artifactId>opengauss-jdbc</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.mysql</groupId>
             <artifactId>mysql-connector-j</artifactId>
+            <scope>provided</scope>
```

Comment:
- My understanding is that this is not enough. If a dependency's maven scope is not `test`, then the dependency will be affected by https://www.apache.org/legal/resolved.html . This requires implementing something like https://github.com/apache/gravitino/issues/4352#issuecomment-2453295083 for `<optional>`, i.e.
```xml
<dependency>
            <groupId>com.mysql</groupId>
            <artifactId>mysql-connector-j</artifactId>
            <scope>provided</scope>
            <optional>true</optional>
</dependency>
```
- The core reason is that MySQL JDBC Driver is a standard GPL. The FOSS exception clause of MySQL JDBC Driver is actually not paid attention to by ASF.